torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -580,8 +585,9 @@ def affine_image_pil(
     return _FP.affine(image, matrix, interpolation=pil_modes_mapping[interpolation], fill=fill)
 
 
-def _affine_bounding_box_xyxy(
+def _affine_bounding_box_with_expand(
```

Comment:
expand was already handled before so this just clarifies the name, right?

Reply:
no, now it handles format -> xyxy conversion inside `_affine_bounding_box_with_expand` and previously it was not the case but we wanted to add clamping there. Basically, using this refactor we can simplify the code for affine-like transforms